package-lock.json

```diff
@@ -1595,37 +1235,14 @@
       "integrity": "sha1-uDx1fIAOaOHW78GjoaE/85/23NI=",
       "dev": true,
       "requires": {
-        "jasmine-node": "jasmine-node@git+https://github.com/kevinsawicki/jasmine-node.git#81af4f953a2b7dfb5bde8331c05362a4b464c5ef",
+        "jasmine-node": "git+https://github.com/kevinsawicki/jasmine-node.git#81af4f953a2b7dfb5bde8331c05362a4b464c5ef",
```

Comment:
By the way, this line is the only change strictly needed in `package-lock.json`. But the other package updates don't hurt.

@aminya's commit includes my entire PR #105 and more. If this PR gets merged, #105 should be considered obsolete and it can be closed (or I will close it if no-one else does).

Reply:
You should not edit machine-generated files like package-lock.json directly

Reply:
Added close hook to the PR description

Reply:
Getting a bit off-topic, but on the contrary, I know how npm's lockfiles are formed, and how they are meant to be read by npm. I read it constantly myself for debugging purposes, and I do edit lockfiles all the time. _(In my opinion, It's extremely useful, especially given how often npm and other package managers makes a lockfile that's slightly wrong.)_

Reply:
@aminya There is no closed-by keyword, you have to add "closes: #105" to your PR.